app.py

```diff
@@ -330,9 +331,18 @@ def clipCollection(image):
     collection = collection.map(clipCollection)
     return collection
 
+# File Parser: GPKG (.gpkg)
+def parse_gpkg_coords_to_ee_geometry(file):
+    # load GPKG file data to fiona collection
+    with fiona.open(fp=file) as f:
+        # convert coordinate data to Google earth engine geometry multipolygon
+        return ee.Geometry.MultiPolygon(
```

Comment:
gee throws in an error after trying to process the returned geometry because it is invalid, ee.geometry.multipolygon exects a certain nesting for polygons, multipolygons and polygons with multiple features (like islands in an archipelago)... etc so i expect that fiona is passing raw coordinates so it may not match gee's nesting structure. I tried restructuring the coordinates but still produced same invalid geometry error